### PR TITLE
General Updates

### PR DESCRIPTION
**Pull request by darvid7**
- update readme, added requirements section & node.js 8.x
- add .idea to gitignore
- change url in package.json

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,5 @@
 # swagger-platform
 Project in development - Open sourced service overlay for SDK management using Swagger-codegen
+
+# Minimum Requirements
+- Node 8.x
```
Comment (PatrickShaw) [inline README.md]: We should add an .nvmrc file

Comment (darvid7) [reply README.md]: Not sure what that is. Do you want to add that?

Comment (PatrickShaw) [reply README.md]: Yeah i'll add it in later

Comment (darvid7) [reply README.md]: Thanks, feel free to merge after you add it

**Pull request merged**